neuro_evolution_ctrnn/brains/i_brain.py

```diff
@@ -140,3 +153,44 @@ def get_input_weight_matrix(self):
 
     def get_output_weight_matrix(self):
         raise NotImplementedError("Support for brain visualizer is not implemented for this brain.")
+
+    @classmethod
+    def get_activation_function_numpy(cls, activation: str) -> Callable[[np.ndarray], np.ndarray]:
```

Comment:
this could also eventually be used by other brains as well, or?

Reply:
Yes, this was also an intention.